monitor/ui/main.py

```diff
@@ -255,6 +257,13 @@ def set_reboot_flag(self):
         """
         self.reboot_flag = True
 
+    def set_monitor_mode(self):
+        if self.monitor_mode == "monitor":
+            self.monitor_mode = "service"
+        else:
+            self.monitor_mode = "monitor"
```

Comment:
For this method it would be nice to set a desired state. So instead of a generic toggle define the signature like this:
```python
def set_monitor_mode(self, service_mode: bool) -> None: ...
```